grpc-sys/build.rs

```diff
@@ -287,6 +287,7 @@ fn bindgen_grpc(mut config: bindgen::Builder, file_path: &PathBuf) {
         .default_enum_style(bindgen::EnumVariation::Rust {
             non_exhaustive: false,
         })
+        .derive_default(true)
```

Comment:
I don't think it's a good idea. Usually grpc provides API to initial the struct correctly. It's not guaranteed zeroed struct is a correct struct.

Reply:
fixed